src/us/kbase/common/utils/CondorUtils.java

```diff
@@ -128,11 +130,11 @@ public static CondorResponse runProcess(String[] condorCommand) throws IOExcepti
             e.printStackTrace();
         }
 
-        if(process.isAlive()){
+        CondorResponse failure = new CondorResponse(stdOutMessage, stdErrMessage, false);
+        if (process.isAlive()) {
```

Comment:
we want this to fail if isn't completed in 30 seconds?

Reply:
yes